README.md

```diff
@@ -21,3 +21,38 @@ cookiecutter gh:Thermondo/django-package-template
 
 Cookiecutter documentation can be found here:
 https://cookiecutter.readthedocs.io/
+
+#### Releasing a package
```

Comment:
I think this section should be on the same level as `Usage` and `Documentation`. Because now it looks like it's a subsection of `Documentation`. And maybe they should all actually be level 2 (`##`) to have a nice underscore.